students/11th/haejunkim/westagram/User/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from .views import SignupView, LoginView
```

Comment:
```suggestion
from .views import (
    SignupView, 
    LoginView
)
```
한 번에 여러 클래스를 import 하는 경우 서로 분리시켜주시는 것을 추천드립니다.